cli/determined_cli/trial.py

```diff
@@ -205,6 +205,46 @@ def kill_trial(args: Namespace) -> None:
                             "of the log (default is all)",
                         ),
                     ),
+                    Arg(
+                        "--agent-id",
```

Comment:
blocking: This needs to match the name in the proto definition (also other fields), so, at a minimum:
```suggestion
                        "--agent-ids",
```
But it would be a bit weird to require arguments like `--agent-ids foo --agent-ids bar`, so maybe you want
```suggestion
                        "--agent-id",
                        dest="agent_ids",
```